python-sdk/example_dags/example_load_file.py

```diff
@@ -253,4 +253,17 @@
     )
     # [END load_file_example_19]
 
+    # [START load_file_example_20]
+    aql.load_file(
+        input_file=File(path="sftp://upload/", conn_id="sftp_conn", filetype=FileType.CSV),
```

Comment:
what will happen if I have multiple files in the upload folder?

Reply:
it will get both the files.

Reply:
@rajaths010494 Can we add this behavior to docs? also, add a test to test this as well?

Reply:
Modify to support prefixes as well.

Reply:
if i have two files abc and def in the upload folder, and I want to upload only abc to  the target table - will the above code work?

Reply:
Now i have modified the code to take prefix now the path should sftp://upload/abc if you want to upload the file abc only.